pkg/virt-api/webhooks/validating-webhook/admitters/vmpool-admitter.go

```diff
@@ -76,6 +78,17 @@ func (admitter *VMPoolAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissio
 	}
 
 	causes := ValidateVMPoolSpec(ar, k8sfield.NewPath("spec"), &pool, admitter.ClusterConfig)
+
+	if ar.Request.Operation == admissionv1.Create {
```

Comment:
Shouldn't it validate FG first before getting into the spec? (similar to VMI admitter)